src/modules/users/users.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { UsersService } from './users.service';
+import { UsersController } from './users.controller';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { UserEntity } from './entities/user.entity';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([UserEntity])],
+  controllers: [UsersController],
+  providers: [UsersService],
```

Comment:
if we just use the service, i think (not for now) we can delete the UserController and the UserModule in the future and just keep the UserService, this change is not part of this scope but just to keep in mind

Reply:
Okay understood not for now!!